gsoc-2020/extension-activity-monitor.md

```diff
@@ -13,31 +13,27 @@ additional useful tool to aid investigating bugs in the extensions or in the Fir
 
 ## Skills Required
 
-An applicant needs familiarity with:
+An applicant needs:
 
-* HTML/JavaScript and the Web API
-* Revision Control Systems (e.g. git and/or mercurial)
+* Familiarity with front-end web development (JavaScript, HTML, CSS).
+* A good understanding of the capabilities of web APIs.
+* Be productive with version control systems (e.g. Git or Mercurial).
+* Be able to communicate clearly in English.
+* Think critically and be able to independently investigate/debug issues.
+* Enjoy working with others, in an open-source setting.
 
-and the following skills:
-
-* Contributing to Open Source
-* Think critically
-* Good reading comprehension
-* Do independent research
-* Enjoy working with others
-* Be productive with little supervision
-* Able to work remotely
+Knowledge of browser extension APIs is a bonus.
 
 ## Project Details
 
-An `activityLog` API is available (in Firefox >= 70) to privileged extensions, the applicant goal is leveraging this API to
+An `activityLog` API is available (in Firefox >= 70) to privileged extensions, the applicant's goal is leveraging this API to
 create a privileged extension which would explore various ways to present this information to the users.
 
-In particular, this project goal is to expore the following areas:
+In particular, this project's goal is to expore the following areas:
 
-- Browser tab (about:extensionmonitor, maybe) that shows all extensions active in all open tabs, extensions altering search, and for each installed extension
-  show activity like network requests and data interactions between content scripts and background scripts.
-- For the current tab, show extensions acting on it (maybe in the Site Information door hanger).
+- An extension page that shows extensions active in all open tabs, extensions altering browser's settings and/or webpages requests,
```

Comment:
>  and/or webpages requests

I don't know what you mean by that. Are you referring to network requests? That is covered by the next bullet point already.

Reply:
the other point is about network request originated from an extension, this was meant to mean webpages requests altered by an extension (basically webRequest and proxy APIs logs)

(I wouldn't be against removing `and/or webpages requests`, in the end it would be implicitly part of the extension APIs activity logs related to all tabs or a specific one)

Reply:
I interpreted "that shows extensions active in all open tabs" as showing the relevant global activities (opposed to tab-specific ones below). Did you actually mean an aggregation of all tab-specific activities?

Does this capture what you meant to write?

> An extension page that shows the active extensions and their effect on the browser, such as altered browser settings or modified network requests.